src/code/z_skin.c

```diff
@@ -1,23 +1,262 @@
 #include "global.h"
+#include "z64skin.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin/func_80137970.s")
+MtxF D_801F5AC0[60];
+UNK_TYPE1 D_801F69C0[0x10];
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin/func_80137B34.s")
+void func_80137970(MtxF* mtx, SkinVtx* skinVertices, Struct_800A598C* arg2, Vtx* verticesDst, Vec3f* arg4) {
+    SkinVtx* skinVtxEntry;
+    Vtx* vtx;
+    Vec3f sp5C;
+    Vec3f sp50;
+    Vec3f sp44;
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin/func_80137EBC.s")
+    sp5C.x = mtx->mf[3][0];
```

Comment:
This is more of a general question since I haven't used `MtxF` all that often. I noticed it has a union of a 2D array, and a struct with the indices named into the 2D array. Which one should we be using, index as a 2D array (what you have here), or use the indices themselves (for example `mf[3][0]` is the same as `zx` I think)? Also do we know if which one we uses affects codegen?